portal/static/js/admin.js

```diff
@@ -1,652 +1,781 @@
-                                  data.status.forEach(function(status) {
-                                      var c = status.consents;
-                                      var a = "", s = "", prevItem = {};
-                                      if (c) {
-                                      c.forEach(function(item) {
-                                          if (!item.consent.deleted && (!prevItem.consent_signed || (prevItem.assessment_status != item.assessment_status)
-                                              || (String(prevItem.consent_signed).substring(0, 10) != String(item.consent.signed).substring(0, 10)))) {
-                                              if (!(/null/.test(item.consent.agreement_url))) {
-                                                var cl = "";
-                                                var sd = tnthDates.formatDateString(item.consent.signed);
-                                                var status = item.assessment_status;
-                                                if (!item.consent.send_reminders) status = "withdrawn";
-                                                switch(String(status).toLowerCase()) {
-                                                    case "completed":
-                                                      cl = "text-success";
-                                                      break;
-                                                    case "withdrawn":
-                                                      cl = "text-muted";
-                                                      break;
-                                                    case "due":
-                                                      cl = "text-warning";
-                                                      break;
-                                                    case "overdue":
-                                                      cl = "text-danger";
-                                                      break;
+    return self.patientsIdList;
+  };
+  /*
+   * currently not being used - client side retrieval of assessment status
+   */
+  AdminTool.prototype.getData = function(requests, callback) {
+      var self = this;
+      if (self.ajaxAborted) {
+          return false;
+      };
+      if (!requests || requests.length === 0) {
+          return false;
+      };
+      var userString = requests.shift();
+      if (!hasValue(userString)) {
+          return false;
+      };
+
+      var tnthDates = self.getDependency("tnthDates");
+      /*
+       *  load the data sequentially
+       *  Note, NO concurrent ajax calls here,
+       *  one request will wait after the previous one has finished
+       */
+      var ajaxRequest = $.ajax ({
+                                type: "GET",
+                                url: "/api/consent-assessment-status",
+                                contentType: "application/json; charset=utf-8",
```

Comment:
Is this necessary? Shouldn't we be setting a default for `contentType` in `ajaxSetup()`?